src/components/slide-carousel/README.md

```diff
@@ -8,24 +8,20 @@ const Slide = require('./Slide').default;
 <div style={ { width: "700px" } }>
     <SlideCarousel contentHeight="250px" initialIndex={3} title="Love & Happiness">
         <Slide>
-            <Logo/>
```

Comment:
None of these components are being imported above, so this was throwing errors on the `/Components` page.